hadoop/src/main/java/org/apache/carbondata/hadoop/util/CarbonVectorizedRecordReader.java

```diff
@@ -128,6 +128,9 @@ private boolean nextBatch() {
     if (iterator.hasNext()) {
       iterator.processNextBatch(carbonColumnarBatch);
       numBatched = carbonColumnarBatch.getActualSize();
+      if (numBatched == 0) {
```

Comment:
This change is not related to binary, suggest to keep it in separate PR

Reply:
ok, removed